theseus_gui/src/components/ui/RunningAppBar.vue

```diff
@@ -1,5 +1,9 @@
 <template>
   <div class="action-groups">
+    <a href="https://github.com/modrinth/theseus/issues" class="link">
+      <BugIcon />
+      <span> Report a bug </span>
+    </a>
```

Comment:
IDK if this is the right move, as a lot of users will report stuff here which is not bugs. I think it would be better to have a get support button and link to our discord channel